custom/lib_portal/models/res_partner.py

```diff
@@ -12,9 +12,12 @@ def _isMember(self):
     date_of_birth = fields.Date(string='Date of Birth', 
                                 default=fields.Date.context_today)
     date_of_death = fields.Date(string='Date of Death') 
-    languages_known = fields.Many2many('res.lang',string='Known Languages')
+    languages_known = fields.Many2many('res.lang', 'partner_lang_rel', 
+                                       string='Known Languages')
+	is_author = fields.Boolean(string='Is a Author', default=False, store=True)
```

Comment:
indentation here seems off

Reply:
missed while committing.will take care in the next commit